src/resources/breaking-changes.md

```diff
@@ -45,46 +45,241 @@ To be notified about future breaking changes, join the [Dart announce][] group.
+accounting for them before the 3.4.0 release.
+
+### Language
+{: .no_toc}
+
+* [The pattern context type schema for cast patterns
+  is now `_` (the unknown type) instead of `Object?`][54640].
+
+[54640]: https://github.com/dart-lang/sdk/issues/54640
+
+### Libraries
+{: .no_toc}
+
+#### `dart:cli`
+
+* **Experimental** **Removed**: [The `waitFor` function][52121]
+  is set to be removed.
+
+[52121]: https://github.com/dart-lang/sdk/issues/52121
+
+#### `dart:html`, `dart:indexed:db`, `dart:svg`, `dart:web_audo`, `dart:web_gl`
+
+- These libraries are now marked as legacy and
+  will see less support in the future.
+  New projects should prefer to use [`package:web`][] and `dart:js_interop`.
+
+[`package:web`]: {{site.pub-pkg}}/web
 
-* [Marked classes belonging to `NativeWrapperClass` as `base`][51896] so that
-  none of their subtypes can be implemented.
+#### `dart:js`
+
+- This library is now marked as legacy and will see less support in the future.
+  Usages should be migrated to `dart:js_interop` and `dart:js_interop_unsafe`.
+  To learn more, check out [`/go/next-gen-js-interop`][].
+
+[`/go/next-gen-js-interop`]: https://dart.dev/go/next-gen-js-interop
+
+#### `dart:js_util`
+
+- This library is now marked as legacy and will see less support in the future.
+  Usages should be migrated to `dart:js_interop` and `dart:js_interop_unsafe`.
+  To learn more, check out [`/go/next-gen-js-interop`][].
+
+[`/go/next-gen-js-interop`]: https://dart.dev/go/next-gen-js-interop
 
 #### `dart:typed_data`
 
-* **Deprecated**: [Unmodifiable view classes for typed data][53218].
+* [The unmodifiable view classes for typed data are deprecated][53128].
+  Instead of using the constructors of these classes, use
+  the new `asUnmodifiableView` methods on typed data lists.
 
+[53128]: https://github.com/dart-lang/sdk/issues/53218
 
-[51896]: https://github.com/dart-lang/sdk/issues/51896
-[53218]: https://github.com/dart-lang/sdk/issues/53218
+### Tools
+{: .no_toc}
 
-{% comment %}
-Create new section from these headers for each release.
-If no changes exist in a section (e.g. Language, `dart:async`, etc.),
-don't include the section header.
+#### Production JavaScript compiler (dart2js)
+
+* You should now specify a format to the `--dump-info` flag
+  of either `binary` or `json`.
+  The `json` format is deprecated and might be removed in a future Dart release.
+
+#### Wasm compiler (dart2wasm)
+
+* **Experimental**: Various `dart compile wasm` CLI arguments have
+  been updated, removed, or replace.
+  To learn more, run `dart compile wasm --verbose --help`.
+
+## 3.3.0
+
+**Tentative**<br>
+The following changes are expected to be included in the 3.3.0 stable release,
+but the final list might change before then.
+To reduce any potential impact of these changes, consider
+accounting for them before the 3.3.0 release.
+
+### SDK
+{: .no_toc}
 
-## Released in x.x.x
+* The following experiments are now retired as they were released in Dart 3.
```

Comment:
Would this count as an Experimental / Removed?
```suggestion
* **Experimental** **Removed**: The following experiments are now retired as they were released in Dart 3.
```

Reply:
Hmm. I guess so? For now, works for me. Definitely makes sense to add Experimental though!

Once I get around to working on the labels we can think more precisely about the categories and how they help users.